flytekit/configuration/__init__.py

```diff
@@ -530,6 +530,7 @@ class S3Config(object):
     backoff: datetime.timedelta = datetime.timedelta(seconds=5)
     access_key_id: typing.Optional[str] = None
     secret_access_key: typing.Optional[str] = None
+    s3_additional_kwargs: typing.Optional[str] = None
```

Comment:
To be passed into S3Config for s3fs, see https://github.com/fsspec/s3fs/blob/main/s3fs/core.py#L203-L204